src/client/src/index.js

```diff
@@ -2,12 +2,12 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import { BrowserRouter } from 'react-router-dom';
 import './index.css';
-import App from './components/App/index';
 import registerServiceWorker from './registerServiceWorker';
+import Login from './components/Content/Login/index.js';
 
 ReactDOM.render(
```

Comment:
Will have to think about this, we want to be able to handle the user opening the app and depending on if they have an open session or not, it would direct them either to / or /login

Reply:
I am currently looking into this.

Reply:
After discussing it with @merekorlowski , I will cancel the changes to the src/client/src/index.js file so that it can be done in another way.